src/ErrorReporting/V1beta1/ErrorGroupServiceClient.php

```diff
@@ -181,18 +189,21 @@ public function __construct($options = [])
             ],
             'retryingOverride' => null,
             'timeoutMillis' => self::DEFAULT_TIMEOUT_MILLIS,
-            'appName' => 'gax',
-            'appVersion' => AgentHeaderDescriptor::getGaxVersion(),
+            'libName' => null,
+            'libVersion' => null,
         ];
         $options = array_merge($defaultOptions, $options);
 
+        if (isset($options['libVersion'])) {
```

Comment:
What do you think about the following on line 205 instead of lines 197-201 here?:

```php
'libVersion' => $options['libVersion'] ?: self::getGapicVersion(),
```